r2/r2/controllers/api.py

```diff
@@ -752,6 +754,9 @@ def POST_leavecontributor(self, container):
         """
         if container and container.is_contributor(c.user):
             container.remove_contributor(c.user)
+            if container.is_moderator_with_perms(c.user, 'access'):
```

Comment:
Alright. I don't think it's really a big deal, but either way it needs to be in a separate commit since it's completely unrelated to the "automatic permissions to successor" stuff.